LayoutTests/imported/w3c/web-platform-tests/scroll-animations/css/animation-timeline-none-expected.txt

```diff
@@ -1,4 +1,4 @@
 
 FAIL Animation with animation-timeline:none holds current time at zero assert_equals: expected "100px" but got "0px"
-FAIL Animation with unknown timeline name holds current time at zero assert_equals: expected "100px" but got "101.9375px"
+FAIL Animation with unknown timeline name holds current time at zero assert_equals: expected "100px" but got "100.203125px"
```

Comment:
This is just a flaky result, no need to commit this.

Reply:
Fixed.